opensrp-anc/src/test/java/org/smartregister/anc/library/interactor/ContactInteractorTest.java

```diff
@@ -108,26 +143,19 @@ public void testFinalizeContactFormInvokesUpdatesPatientRepositoryWithCorrectPar
         details.put(DBConstantsUtils.KeyUtils.BASE_ENTITY_ID, DUMMY_BASE_ENTITY_ID);
         details.put(DBConstantsUtils.KeyUtils.NEXT_CONTACT, "1");
 
-        final List<String> contactDates = new ArrayList<>();
-        contactDates.add("10");
-        contactDates.add("20");
-        contactDates.add("30");
-        contactDates.add("40");
-
-
         PowerMockito.mockStatic(AncLibrary.class);
         PowerMockito.mockStatic(PatientRepositoryHelper.class);
 
 
         PowerMockito.when(PatientRepositoryHelper.getWomanProfileDetails(DUMMY_BASE_ENTITY_ID)).thenReturn(details);
-        PowerMockito.when(AncLibrary.getInstance()).thenReturn(AncLibrary);
-        PowerMockito.when(AncLibrary.getAncRulesEngineHelper()).thenReturn(ancRulesEngineHelper);
-        PowerMockito.when(AncLibrary.getDetailsRepository()).thenReturn(detailsRepository);
-        PowerMockito.when(AncLibrary.getPreviousContactRepositoryHelper()).thenReturn(previousContactRepositoryHelper);
+        PowerMockito.when(org.smartregister.anc.library.AncLibrary.getInstance()).thenReturn(ancLibrary);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.anc.library.AncLibrary.getInstance' due to existing import 'org.smartregister.anc.library.AncLibrary'](https://app.codacy.com/app/OpenSRP/opensrp-client-anc/pullRequest?prid=4499997)